build/ci/evergreen.yml

```diff
@@ -992,7 +979,22 @@ tasks:
     commands:
       - func: "clone"
       - func: "generate docs"
-      - func: "upload docs"
+      - command: archive.targz_pack
```

Comment:
targz all the docs and upload them as a single file to make the downloading of the whole docs easier